src/tasks/leveling.ts

```diff
@@ -856,7 +857,10 @@ export const LevelingQuest: Quest = {
     {
       name: "Get Rufus Quest",
       completed: () => get("_shadowAffinityToday") || !have($item`closed-circuit pay phone`),
-      do: () => chooseQuest(() => 2),
+      do: (): void => {
+        chooseQuest(() => 2);
+        if (holiday().includes("April Fool's Day")) visitUrl("questlog.php?which=7");
```

Comment:
This is fine and all, but we will still rely on mafia automating the artifact quest, and there's no guarantee that mafia can do so with all the salad (without breaking) [or even correctly set the `_shadowAffinityToday` flag?]

It's probably a rabbit hole we'll go down if we want to fully support April Fool's, and that's not something I'd really wanna get into. But for now I think this is probably acceptable